fintual-oss/paraffin-back
# feat(seeds): replace seed file for the final one

# Base PR
- Definite seed file.

# Context
- With this PR, we have the seeds ready for the presentation.

# Changelog
- Replace seed file.

# QA
- Launch Rails console
- Run `Command.perform`
- Check that it returns...
